src/jenkins/BuildManifest.groovy

```diff
@@ -49,6 +63,14 @@ class BuildManifest implements Serializable {
             this.getArtifactRoot(jobName, buildNumber)
         ].join('/')
     }
-}
-
 
```

Comment:
I believe the manifest contains the product name, doesn’t it?

Reply:
Yes. I've updated the `getPackageName()` by using `this.buid.getFilename()`.